tony-history-server/app/controllers/JobsMetadataPageController.java

```diff
@@ -0,0 +1,42 @@
+package controllers;
+
+import com.typesafe.config.Config;
+import java.util.ArrayList;
+import java.util.List;
+import javax.inject.Inject;
+import models.JobMetadata;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.Path;
+import play.Logger;
+import play.Logger.ALogger;
+import play.mvc.Controller;
+import play.mvc.Result;
+
+import static utils.HdfsUtils.*;
+import static utils.ParserUtils.*;
+import static utils.SecurityUtils.*;
+
+
+public class JobsMetadataPageController extends Controller {
```

Comment:
How about JobsListPageController? JobsMetadata make it sound like you are showing metadata on a particular job rather than listing all jobs.

Reply:
Hence it's 'JobsMetadata': metadata of multiple jobs, not one.

Reply:
I agree 'JobsMetadata' is an accurate name, but I think 'JobsList' more directly captures the purpose of the page -- which lists all the TonY jobs that have finished or are running. The 'List' part emphasizes that the page shows info about multiple jobs.

Minor point, so I leave it up to you to pick a name.